examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1107,7 +1144,34 @@ class GLTFWriter {
 
 			if ( image.width > options.maxTextureSize || image.height > options.maxTextureSize ) {
 
-				console.warn( 'GLTFExporter: Image size is bigger than maxTextureSize', image );
+				ctx.drawImage( image, 0, 0, canvas.width, canvas.height );
```

Comment:
Something doesn't look right here, this block doesn't warn, and the `image.width > options.maxTextureSize` block below is unreachable.